tests/test_adjoint_jacobian.py

```diff
@@ -125,14 +122,12 @@ def test_not_expval(self, dev):
             qml.RX(0.1, wires=0)
             qml.state()
 
-        if device_name == "lightning.kokkos" and ld._CPP_BINARY_AVAILABLE:
+        if device_name == "lightning.kokkos":
             message = "Adjoint differentiation does not support State measurements."
-        elif device_name == "lightning.gpu" and ld._CPP_BINARY_AVAILABLE:
+        elif device_name == "lightning.gpu":
             message = "Adjoint differentiation does not support State measurements."
-        elif ld._CPP_BINARY_AVAILABLE:
-            message = "This method does not support statevector return type."
         else:
-            message = "Adjoint differentiation method does not support measurement StateMP"
+            message = "This method does not support statevector return type."
```

Comment:
As with earlier, is this error message descriptive enough in the main implementation?

Reply:
Idem.